propagation/propagation.go

```diff
@@ -1,119 +1,59 @@
 package propagation
 
 import (
-	"encoding/base64"
-	"encoding/json"
 	"fmt"
-	"net/url"
-	"strings"
 )
 
-// assumes a header of the form:
+// Default to using the Honeycomb Trace Context header.
+const defaultTracePropagationHTTPHeader = TracePropagationHTTPHeader
```

Comment:
I'm not sure what this line is for. Perhaps it's intended for a future PR?

Reply:
That's correct - I'm going to remove it from this PR though - meant to remove it before pushing.